project/plugins.sbt

```diff
@@ -1,4 +1,4 @@
 resolvers += Resolver.bintrayRepo("bbp", "nexus-releases")
 
-addSbtPlugin("ch.epfl.bluebrain.nexus" % "sbt-nexus"     % "0.10.13")
+addSbtPlugin("ch.epfl.bluebrain.nexus" % "sbt-nexus"     % "0.11.0")
```

Comment:
Don't bump to this version yet, it drops the agents for monitoring which is not yet supported in KG.